plugins/modules/ios_vlans.py

```diff
@@ -876,7 +871,7 @@
 #         }
 #     ]
 
-# Using Parsed (configuration: True)
+# Using Parsed Vlan configuration only
```

Comment:
Can we add an example for `parsed` that show how to deal with both configuration stanzas in the same task?